desktop/libs/indexer/src/indexer/api3.py

```diff
@@ -239,7 +240,7 @@ def guess_field_types(request):
 
       if file_format['format']['hasHeader']:
         sample = csv_data[1:5]
-        column_row = csv_data[0]
+        column_row = [re.sub('[^0-9a-zA-Z]+', '_', col) for col in csv_data[0]]
```

Comment:
Maybe declare this regex above and use the variable in diff places? Easier to modify it at one place if needed

Reply:
Actually, external tables (we are using in remote file option) support some more charsets so I am not using this for the remote_file option as there are more things to understand is this needed or not in other options. In the future, if needed we can modify it accordingly?